test/libsolidity/util/BytesUtils.cpp

```diff
@@ -206,10 +218,33 @@ string BytesUtils::formatString(bytes const& _bytes, size_t _cutOff)
 	return os.str();
 }
 
+std::string BytesUtils::formatFixedPoint(bytes const& _bytes, bool _signed, size_t _fractionalDigits)
+{
+	string value;
+	bool negative = false;
+	if (_signed)
+	{
+		s256 signedValue{u2s(u256{_bytes})};
+		if (signedValue < 0)
+		{
+			negative = true;
+			signedValue = -signedValue;
+		}
+		value = signedValue.str();
+	}
+	else
+		value = u256{_bytes}.str();
```

Comment:
Why does `formatUnsigned` use `fromBigEndian` ( https://github.com/ethereum/solidity/pull/11682/files#diff-7e30501636905a85639f7d966bf02472db2c18803f9b2532b7d9b11f5cfcbbecR141 )? Is `_bytes` in binary or in decimal / hex?

Reply:
Ok, I changed it now to use `fromBigEndian` as done in `formatUnsigned`. `_bytes` is binary.